app/uk/gov/hmrc/economiccrimelevyaccount/views/AccountView.scala.html

```diff
@@ -30,5 +34,101 @@
 
     <h1 class="govuk-heading-xl govuk-!-margin-bottom-0">@messages("account.heading")</h1>
     <span class="govuk-caption-m">@messages("account.eclReferenceNumber", eclRegistrationReference)</span>
+    <p>
+
+
+        <br>
+
+    <div class="flex-container govuk-grid-row">
+    <!-- RETURNS CARD -->
+        <div class="card" id="aft-overdue-charges">
+            <div class="card-body">
+                <h2 class="card-heading govuk-heading-m">
+                    @messages("account.return.card.title")
+                </h2>
+                <p class="card-sub-heading bold govuk-body-m">
+                    @if(obligationDetails.nonEmpty && obligationDetails.get.status == Open){
+                        @defining(obligationDetails.get) { details =>
```

Comment:
Pattern matching on this would be cleaner I think

Reply:
Done.